package.json

```diff
@@ -39,6 +39,7 @@
     "@types/react": "^18.0.0",
     "@types/url-parse": "^1.4.3",
     "@typescript-eslint/eslint-plugin": "^6.13.1",
+    "@typescript-eslint/parser": "^7.0.2",
```

Comment:
The following packages added in this PR are already bundled in the repo:

- @typescript-eslint/parser
- eslint-plugin-react
- eslint-plugin-react-hooks

Do they need to be explicitly added here?

Reply:
this was out of habit from using pnpm. removed this since they're dependencies of the next config https://github.com/vercel/next.js/blob/canary/packages/eslint-config-next/package.json#L15